src/components/TabStepper/TabStepper.stories.tsx

```diff
@@ -146,15 +160,60 @@ export const TabStepperOrientation = {
       </TabStepper>
     );
   },
-
   name: 'Orientation',
-
   parameters: {
     controls: { include: ['orientation'] },
   },
 };
 
-export const TabStepperStatuses = {
+export const TabStepperDisabled: Story = {
+  render: () => {
+    const [key, setKey] = React.useState<TabKey>('tab-1');
+
+    const items: TabStepItem[] = [
+      {
+        id: 'tab-1',
+        title: 'Tab label',
+        subtitle: 'Secondary information line',
+        status: 'done',
+      },
+      {
+        id: 'tab-2',
+        title: 'Tab label',
+        subtitle: 'Secondary information line',
```

Comment:
Lets write here "This is disabled for this scenario"

```suggestion
        subtitle: 'This is disabled for this scenario',
```